gulp-tasks/analyze-properties.js

```diff
@@ -1,14 +1,24 @@
 const gulp = require('gulp');
 
+const build = require('./build');
+
 const AnalyseBuildForProperties = require('./utils/analyse-properties');
 
-gulp.task('analyze-properties:run', async () => {
+const runAnalyzeProperties = async () => {
   const analysisTool = new AnalyseBuildForProperties();
   const results = await analysisTool.run();
   results.forEach((entry) => {
     analysisTool.printDetails(entry);
   });
-});
+};
+runAnalyzeProperties.displayName = 'analyze-properties:run';
+// GULP: Is this exposed to the CLI?
```

Comment:
Yes it is (this is a somewhat weird task I concocted to try and spot missed minification opportunities).